test-cases/longevity/longevity-large-collections-48h.yaml

```diff
@@ -0,0 +1,19 @@
+# 48 hours
+test_duration: 2880
+prepare_write_cmd: "cassandra-stress write cl=ALL n=1000000  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..200200300 -log interval=15"
+stress_cmd: [
+"JVM_OPTION='-Xms8033M -Xmx8033M -Xmn400M' cassandra-stress user profile=/tmp/large_collections.yaml ops'(insert=3, read1=1)' cl=QUORUM duration=2860m -port jmx=6868 -mode cql3 native -rate threads=10"
+             ]
+
+n_db_nodes: 4
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 5
+nemesis_during_prepare: false
```

Comment:
no need for prepare stage at all

Reply:
I will try to remove the prepare stage.

Last time I saw one nemesis failed for lack of keyspace1.standard1, then I added the prepare stage.